### DISCUSSION TITLE
如何使用？

### DISCUSSION THREAD

**Discussion by Matthew-Harris-36** [cangzhang/ChampR#299] [General]
国服客户端，v2.0.2-b6压缩文件里只有一个champ-r客户端，而且打开后不像旧版本还可以手动选择LOL安装路径，变成了自动检测，但是一直检测不到啊，就没办法正常使用。我也尝试分别把exe文件移到LOL的tecent game\英雄联盟\TCLS和f:\tecent game\英雄联盟\LeagueClient路径下面，都是一样的问题。我查看旧版本压缩文件里还会有一个LeagueClient.exe，请问这个是干嘛的啊，英雄联盟文件不是腾讯下载时用的吗？

**Comment by cangzhang** [comment]
试试最新的 b7 版本？

**Comment by Matthew-Harris-36** [comment]
b7版本我放到其它位置，就一直检测不到LOL位置。放到f:\tecent game\英雄联盟\LeagueClient路径下，第一次运行可以检测到LOL，但一直apply中，结束不了；兼容性设置，以管理员权限启动，再次启动程序，就一直loading。所以正确的用法是怎样的？你讲具体点啊，现在的中文文档使用教程肯定过时了，麻烦更新下。

**Comment by cangzhang** [comment]
检查下你的杀毒软件有没有禁用 ChampR powershell 执行。
一直 apply 中是正常的，多等几分钟就会把选中的数据源所有出装导入。
想验证功能就只选一个数据源，多等几分钟试试
